Source/Core/DolphinQt/NetPlay/ClickBlurLabel.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright 2025 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "ClickBlurLabel.h"
+
+#include <QMouseEvent>
+#include <QVBoxLayout>
+
+#include "Common/CommonTypes.h"
+#include "Common/Random.h"
+
+ClickBlurLabel::ClickBlurLabel(QWidget* parent)
+    : QWidget(parent), m_stack(new QStackedWidget(this)), m_normal_label(new QLabel(this)),
+      m_blurred_label(new QLabel(this))
+{
+  setCursor(Qt::PointingHandCursor);
+
+  auto* layout = new QVBoxLayout(this);
+  layout->setContentsMargins(0, 0, 0, 0);
+  layout->addWidget(m_stack);
+
+  auto* blur = new QGraphicsBlurEffect(m_blurred_label);
```

Comment:
Maybe a comment about `QGraphicsBlurEffect` causing a delay to explain why `QStackedWidget` is being used?